tests/integration/components/reminders-list-test.js

```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test } from 'ember-qunit';
```

Comment:
This is all boilerplate code - lets add some sort of assertion that indicates you know what this file does. In this case I would at least test that there is an h1, and 5 elements with the class of '.spec-reminder-item'.